src/arrays.js

```diff
@@ -1,3 +1,5 @@
+import is from 'core-js-pure/es/object/is';
```

Comment:
Please don't add this ugly commonjs
https://unpkg.com/browse/core-js-pure@3.6.4/es/object/is.js

Better host this code as part of shallow-equal
https://github.com/es-shims/object-is/blob/master/index.js

Reply:
Or host the code from this module
https://unpkg.com/fbjs@1.0.0/lib/shallowEqual.js

Reply:
fbjs is not intended for use outside of facebook. 

`object-is` looks better regarding download count (which reduces probability of higher app-bundle size).

With regards to 

> Please don't add this ugly commonjs

I would remind you to be respectful. What is "ugly" or not is entirely subjective.

Reply:
Sorry I was not clear. I meant host the code in this package please. I pushed this module to have esm support and your changes makes it not compatible again.

Reply:
> I pushed this module to have esm support and your changes makes it not compatible again.

Got you covered: https://github.com/es-shims/object-is/pull/7

Reply:
I'm sure Jordan will not accept PR. He is not a fan of `module` field.

Reply:
I'm interested to learn issues with the `module` field. I don't think you're representing his view in good faith by characterizing it as "being a fan".

Reply:
It's not compatible with node.js modules implementations and he as a member of committee is well aware about it.

Is this better?

Reply:
Well you were completely right. Not about the `module` field (no objects were raised against that) but because of the way they responded.